lib/src/growth_book_sdk.dart

```diff
@@ -8,15 +8,15 @@ abstract class SDKBuilder {
   SDKBuilder({
     required this.apiKey,
     required this.hostURL,
-    required this.attributes,
+    this.attributes,
```

Comment:
`attributes` should not be null. Instead, you can initialize this with a default value.
```dart
  SDKBuilder({
    required this.apiKey,
    required this.hostURL,
    this.attributes = const {},
  })
```

Reply:
I see in the super constructor is nullable, why attributes from SDKBuilder should not be null? What's the impact when we send null or empty map?

Reply:
The core FeatureEvaluator evaluates the condition against the  `attributes` and the `condition`. If  `attributes` are null then we can not evaluate the condition. It's better to have empty in place of null `attributes`. 

make sure it's formatted before you push code. You can ensure it by `flutter format .`. You can fire this from root.

LGTM.